Gemfile.lock

```diff
@@ -2,51 +2,50 @@ PATH
   remote: .
   specs:
     nullifyable (0.1.0)
-      activerecord (>= 4.0)
+      activerecord (>= 6.1)
```

Comment:
@fesilvajacquier Why do we need to change this dependency? I'm OK upgrading dev dependencies, but is there a reason to drop support for rails 5.x?